### ISSUE TITLE
Support for release and pass on switches

### ISSUE THREAD

**Issue by AnthonyLloyd** [humhei/FCSWatch#26]
This does seem to be very fast, thanks.

Would it be possible to add a config to compile as release and for a specific framework?

Also would it be possible to pass on switches e.g. -- --myswitch a.

Then it would have all the functionality of dotnet watch.

**Comment by humhei** [comment]
> Also would it be possible to pass on switches e.g. -- --myswitch a.

May be ref https://github.com/humhei/FCSWatch/pull/14
Pull requested is merged but not documented now

**Comment by AnthonyLloyd** [comment]
Great thanks. Is release and framework possible?

**Comment by humhei** [comment]
https://github.com/humhei/FCSWatch/pull/14/files#diff-7c4e0296b787415cc49b9fc813a3f6c2R82
--framework should be easy to implement
I currently use the prefer framework(netcoreapp > net framework)

**Comment by humhei** [comment]
But i am not clear with the use case of --release..
It will slow down compiling time.

And also has conficts to the debuggable feature (Which i think is more important)

**Comment by AnthonyLloyd** [comment]
yes, but would speed up run time. Could be important for running Expecto.
adding a --release switches that adds --configuration Release looks possible.
I could put in PR if you think ok.

**Comment by humhei** [comment]
OK......
Always happy to receive PR

**Comment by AnthonyLloyd** [comment]
Done

**Comment by tylerhartwig** [comment]
Closed with #27

**Comment by AnthonyLloyd** [comment]
Smart. Thank you.

**Comment by humhei** [comment]
Released as 0.7.8

**Comment by humhei** [comment]
@AnthonyLloyd 
Seems --configuration release is not a good idea
https://github.com/humhei/FCSWatch/tree/master/tests/datas
TestLib2/Library.fs
```fsharp
namespace TestLib2

module Say =
    let hello2 name =
        printfn "He %s " name

    let fromLib2 = "91569955558000900     ssss"
```
```fsharp

[<EntryPoint>]
let main _ =

    //printfn "Hell world from resource %s" resource

    printfn "9999                   %s" (Say.fromLib2)

    /// simulate server
    Console.ReadLine()
    |> ignore

    0 // return an integer exit code
```


###  When compile in release configuration
Say.fromLib2 is compiled to a constant `91569955558000900     ssss`

By this way, If we updated `TestLib2/Library.fs` , 
The program wouldn't change any more

```csharp
using System;
using System.IO;
using Microsoft.FSharp.Core;

// Token: 0x02000002 RID: 2
[CompilationMapping(SourceConstructFlags.Module)]
public static class Test
{
	// Token: 0x06000001 RID: 1 RVA: 0x00002050 File Offset: 0x00000250
	[EntryPoint]
	public static int main(string[] _arg1)
	{
		PrintfFormat<FSharpFunc<string, Unit>, TextWriter, Unit, Unit> format = new PrintfFormat<FSharpFunc<string, Unit>, TextWriter, Unit, Unit, string>("9999                   %s");
		PrintfModule.PrintFormatLineToTextWriter<FSharpFunc<string, Unit>>(Console.Out, format).Invoke("91569955558000900     ssss");
		string text = Console.ReadLine();
		return 0;
	}
}

```



### When compile in debug configuration
Say.fromLib2 is compiled to a variable

By this way, If we updated `TestLib2/Library.fs` , 
The program will be updated
```csharp
using System;
using System.IO;
using Microsoft.FSharp.Core;
using TestLib2;

// Token: 0x02000002 RID: 2
[CompilationMapping(SourceConstructFlags.Module)]
public static class Test
{
	// Token: 0x06000001 RID: 1 RVA: 0x00002050 File Offset: 0x00000250
	[EntryPoint]
	public static int main(string[] _arg1)
	{
		FSharpFunc<string, Unit> fSharpFunc = ExtraTopLevelOperators.PrintFormatLine<FSharpFunc<string, Unit>>(new PrintfFormat<FSharpFunc<string, Unit>, TextWriter, Unit, Unit, string>("9999                   %s"));
		string fromLib = Say.fromLib2;
		fSharpFunc.Invoke(fromLib);
		string text = Console.ReadLine();
		return 0;
	}
}

```
